src/algorithm/branching.jl

```diff
@@ -203,7 +284,7 @@ function branch(
     end
     scores = Dict(
         candidate =>
-            branching_score(state, candidate, parent_result, config) for
+            branching_score(state, form, candidate, parent_result, parent_node, config) for
```

Comment:
Nit: switch the order of the `parent_result` and `parent_node` arguments (this entails changing the signatures above too):

```suggestion
            branching_score(state, form, candidate, parent_node, parent_result, config) for
```